src/main/java/school/lemon/changerequest/java/employees/Accountant.java

```diff
@@ -0,0 +1,18 @@
+package school.lemon.changerequest.java.employees;
+
+/**
+ * Created by Yaroslav Pavlinskiy on 06.01.2017.
+ */
+public class Accountant extends Employee {
+    Accountant(String name) {
+        super(name);
+    }
+
+    public double calculateSalaryForEmployee(Employee e) {
+        if(e instanceof Programmer || e instanceof Manager || e instanceof Accountant)
```

Comment:
1. Using `instanceof ` isn't a good practice and should be avoided if it's possible. I do not see any reason for such check here.
2. The main idea was to calculate salary for a array/list of employees. They should be passed in constructor/setter method. So we'll have some kind of responsibility. This Accountant is responsible for a list/array of employees and calculates their salary.